cmd/kops/get_cluster.go

```diff
@@ -215,6 +214,100 @@ func clusterOutputJson(clusters []*api.Cluster, out io.Writer) error {
 	return nil
 }
 
+// TODO refactor all output JSON methods to use this
```

Comment:
Can we create two helpers:

`func marshalMultipleToYaml(w io.Writer, obj runtime.Object...) error {`

`func marshalMultipleToJson(w io.Writer, obj runtime.Object...) error {`

Should make this simpler